v2/websocket/transport.go

```diff
@@ -97,6 +103,36 @@ func (w *ws) Done() <-chan error {
 	return w.quit
 }
 
+// listen for write requests and perform them
+func (w *ws) listenWriteChannel() {
+	for {
+		select {
+		case <- w.quit: // ws closed
+			return
+		case message := <- w.writeChan:
+			err := w.ws.WriteMessage(websocket.TextMessage, message)
```

Comment:
err is not handled

Reply:
Oppps, good spot. This line needs to removed completely as `ws.Write` is used instead a few lines down